Source/WebCore/rendering/style/RenderStyleInlines.h

```diff
@@ -964,6 +965,21 @@ constexpr bool RenderStyle::preserveNewline(WhiteSpace mode)
     return mode != WhiteSpace::Normal && mode != WhiteSpace::NoWrap;
 }
 
+inline bool RenderStyle::determineEffectiveSkipsContent(const Element* element) const
```

Comment:
Can we make this a static function? We shouldn't add this method to the `RenderStyle` class.

Reply:
You meant a global function, right? Done.